src/deploy/NVA_build/rsyslog.conf

```diff
@@ -49,6 +49,21 @@ $OmitLocalLogging off
 # Logging much else clutters up the screen.
 #kern.*                                                 /dev/console
 
+if $msg contains '{"op":"PUT"' then {
```

Comment:
These are the only actions we want to log?

Reply:
I think so for now we are logging these ops only.